### ISSUE TITLE
new front end

### ISSUE THREAD

**Issue by mathieulagrange** [mathieulagrange/explanes.py#185]
info = {
  'name': 'demo',
  'purpose': 'hello world of the doce package',
  'author': 'mathieu Lagrange',
  'address': 'mathieu.lagrange@ls2n.fr'
  }

plan = {
  'nn_type': ['cnn', 'lstm'],
  'n_layers': np.arange(2, 10, 3),
  'learning_rate': [0.001, 0.0001, 0.00001],
  'dropout': [0, 1]
  }

metrics = {
  'accuracy': [np.mean, '%+', np.std],
  'duration': [np.mean, '*-']
}

def step(setting, experiment):
  setting_metrics = {
    'accuracy': np.random.randn(10),
    'duration': np.random.randn(10)
    }
  return setting_metrics

experiment = doce.Experiment(
  info = info,
  func=step,
  plans = {'plan':plan},
  metrics = metrics,
  paths = {'output': '/tmp/'+info['name']+'/'})

if __name__ == "__main__":
  experiment.cli()
